content/applications/finance/fiscal_localizations/vietnam.rst

```diff
@@ -2,55 +2,311 @@
+   - `Legal notice <https://www.sinvoice.vn/2020/11/hdsd-cach-gui-thong-bao-phat-hanh-hoa-don-dien-tu-qua-mang.html>`_
+
+.. _vietnam/sinvoice/setup:
+
+Set-up
+------
+
+.. _vietnam/sinvoice/setup/sinvoice:
+
+Configuration in SInvoice
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _vietnam/sinvoice/setup/sinvoice/registration:
+
+SInvoice registration
+*********************
+
+Log into SInvoice_. Use the :guilabel:`Username` and :guilabel:`Password` used to register for
+the account.
+
+.. note::
+   To create an account, contact :guilabel:`SInvoice` directly on the portal.
+
+.. _vietnam/sinvoice/setup/sinvoice/template:
+
+Invoice template creation
+*************************
+
+#. On the left side of the overview page, click :guilabel:`Create business information` menu under
+   :guilabel:`Release management`. In the :guilabel:`Update key information` step, fill in the
+   following fields and other optional information, and click :guilabel:`Update`.
+
+   - :guilabel:`Unit name`
+   - :guilabel:`Address`
+   - :guilabel:`Contact person`
+   - :guilabel:`Type of representative documents`
+
+#. In the :guilabel:`Look up digital certificate` step, click :guilabel:`Add new` to add a digital
+   certificate. Choose the :guilabel:`Branch/Enterprise` and
+   :guilabel:`Type of digital certificate` and required field for each type needed. Once all the
+   required information is filled, click :guilabel:`Generate key pair` and then :guilabel:`Record`
+   to save.
+
+#. In the :guilabel:`Manage invoice templates` step, add a new :guilabel:`Invoice template`. Choose
+   the :guilabel:`Invoice type`, :guilabel:`Invoice template code`,
+   :guilabel:`Invoice template name` and other optional information to set the template, and click
+   :guilabel:`Update`.
+
+.. _vietnam/sinvoice/setup/sinvoice/symbol:
+
+Invoice symbol creation
+***********************
+
+On the left side of the main screen, click :guilabel:`Invoice symbol` menu under
+:guilabel:`Release management`. Click :guilabel:`Add new` and choose the
+:guilabel:`Invoice template`, :guilabel:`Status` and add :guilabel:`Invoice symbol`. Status must be
+:guilabel:`Active` to activate the symbol. Choose the remaining options applicable, and click
+:guilabel:`Record` to save.
+
+.. image:: vietnam/sinvoice-set-up-symbol.png
+   :alt: SInvoice symbol set-up
+
+.. _vietnam/sinvoice/setup/sinvoice/notice:
+
+Legal notice issuance
+*********************
+
+#. On the eTax-electronic-tax-system_ page, click :guilabel:`Business` and :guilabel:`Login`.
+
+   .. note::
+      To create an account, click :guilabel:`Register` instead of login.
+
+#. After a successful login, click :guilabel:`Online declaration` under :guilabel:`Tax declaration`
+   menu. On the :guilabel:`Online declaration` menu, select `TB01/AC - Thông báo phát hành hóa đơn` and
+   click :guilabel:`Continue`.
+
+   .. Note::
+      The business needs to register for `TB01/AC - Thông báo phát hành hóa đơn` to select it.
```

Comment:
Hello~
On the government portal, the business should have registered for `TB01/AC - Thông báo phát hành hóa đơn` so that it appears on the selection list to choose from. If the business did not apply, the option will not exist for the user to choose.

I have updated the explanation accordingly, thank you